<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>socius</class>
 <widget class="QMainWindow" name="socius">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>socius</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar {
border-radius:0px;
background-color:rgba(255,255,255,220);
border:1px solid white;
}

QProgressBar:chunk {
border-radius:0px;
background-color:rgba(13, 105, 148,235);
margin-top:1px;
margin-bottom:1px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
 height:20px;
 }

QTabWidget::tab-bar {
left:9px;
background-color:transparent;
}

QTabWidget::pane {
left:0px;
background-color:transparent;
}

QTabWidget#IRCframe {
background-color:rgb(0,0,0,45%);
}

 QTabBar::tab:selected {
				width:120px;
     border:1px solid rgb(0,0,0,45%);
     border-bottom-color: #C2C7CB; /* same as pane color */
border-top-left-radius:8px;
border-top-right-radius:8px;
 }

 QTabBar::tab:!selected {
width:120px;
     border:1px solid rgb(0,0,0,45%);
     margin-top: 2px; /* make non-selected tabs look smaller */
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);

border-top-left-radius:8px;
border-top-right-radius:8px;
 }






QWidget#centralWidget {
border-top-left-radius:10px;
border-top-right-radius:10px;
}

QTreeWidget#serverTree {
border-top-left-radius:0px;
border-bottom-left-radius:0px;
border-top-right-radius:5px;
border-bottom-right-radius:5px;
}

QToolButton {
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:5px;
border-top-right-radius:5px;
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(171, 171, 171, 214), stop:1 rgba(255, 255, 255, 203));
}

QToolButton:hover {
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:5px;
border-top-right-radius:5px;
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(98, 98, 98, 206), stop:1 rgba(183, 222, 255, 156));
color:white;
}

QPushButton {
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:5px;
border-top-right-radius:5px;
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(171, 171, 171, 214), stop:1 rgba(255, 255, 255, 203));
}

QPushButton:hover {
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:5px;
border-top-right-radius:5px;
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(175, 175, 175, 214), stop:1 rgba(183, 222, 255, 203));
color:white;
}

QPushButton:checked {
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;
border-top-left-radius:5px;
border-top-right-radius:5px;
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(98, 98, 98, 206), stop:1 rgba(183, 222, 255, 156));
color:white;
}

QPushButton:!checked {
height:18px;
margin-top:2px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="myCloud">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>5</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sync Layer</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="actionEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>25</y>
      <width>566</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgba(231, 231, 231, 150);
border-top-left-radius:0px;
border-bottom-left-radius:5px;
border-top-right-radius:0px;
border-bottom-right-radius:0px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="IRCframe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>55</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTreeWidget" name="treeWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>136</width>
       <height>441</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(40, 40, 40, 255), stop:1 rgba(98, 98, 98, 255))</string>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>0</y>
       <width>603</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:none;
border:1px solid qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(40, 40, 40, 255), stop:1 rgba(98, 98, 98, 255));</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>598</x>
       <y>19</y>
       <width>139</width>
       <height>390</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(40, 40, 40, 255), stop:1 rgba(98, 98, 98, 255));</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>19</y>
       <width>463</width>
       <height>403</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>598</x>
       <y>409</y>
       <width>138</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>236</x>
       <y>421</y>
       <width>501</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(40, 40, 40, 255), stop:1 rgba(98, 98, 98, 255));
border-top-left-radius:0px;
border-bottom-left-radius:0px;
border-top-right-radius:0px;
border-bottom-right-radius:0px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="activityStream_3">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>419</y>
       <width>102</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Click Here To Change Your Nick on</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:0px;
border:1px solid qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(40, 40, 40, 255), stop:1 rgba(98, 98, 98, 255));
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>textEdit_2</zorder>
    <zorder>lineEdit_3</zorder>
    <zorder>progressBar_2</zorder>
    <zorder>activityStream_3</zorder>
    <zorder>treeWidget_2</zorder>
    <zorder>listWidget_2</zorder>
    <zorder>lineEdit_4</zorder>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>740</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,45%);
border-radius:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="scriptsPortal">
    <property name="geometry">
     <rect>
      <x>553</x>
      <y>5</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Debug Window</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bkgImg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>740</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Projects/gather/images/backgrounds/02494_hachinohesunrise_1024x768.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mediaPortal">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>5</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Media Portal</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="activityStream">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>5</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Activity Stream</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolsButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>5</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/imgs/images/default/tools.png</normaloff>:/imgs/images/default/tools.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="chatPortal">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>5</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chat Portal</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="actionButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>25</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
border-bottom-right-radius:5px;</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::MenuButtonPopup</enum>
    </property>
   </widget>
   <widget class="QFrame" name="webFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>55</y>
      <width>740</width>
      <height>439</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>740</width>
       <height>439</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="authWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>55</y>
      <width>739</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <zorder>bkgImg</zorder>
   <zorder>label_4</zorder>
   <zorder>myCloud</zorder>
   <zorder>actionEdit</zorder>
   <zorder>IRCframe</zorder>
   <zorder>scriptsPortal</zorder>
   <zorder>mediaPortal</zorder>
   <zorder>activityStream</zorder>
   <zorder>toolsButton</zorder>
   <zorder>chatPortal</zorder>
   <zorder>actionButton</zorder>
   <zorder>webFrame</zorder>
   <zorder>authWidget</zorder>
  </widget>
  <action name="actionAccounts">
   <property name="text">
    <string>Accounts</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
